packages/pages/src/dev/dev.ts

```diff
@@ -4,6 +4,39 @@ import runSubProcess from "../util/runSubprocess.js";
 import { devServerPort } from "./server/middleware/constants.js";
 import { autoYextInit } from "./server/autoInit.js";
 import open from "open";
+import { ProjectFilepaths } from "../common/src/project/structure.js";
+
+interface DevArgs extends Pick<ProjectFilepaths, "scope"> {
+  local?: boolean;
+  "prod-url"?: boolean;
+  "open-browser": boolean;
+  noInit?: boolean;
+  scope?: string;
+  noGenFeatures?: boolean;
+  noGenTestData?: boolean;
+}
+
+const handler = async ({
+  local,
+  "prod-url": useProdURLs,
+  "open-browser": openBrowser,
+  noInit,
+  scope,
+  noGenFeatures,
+}: DevArgs) => {
+  if (!noInit) {
+    await autoYextInit();
+  }
+  if (!noGenFeatures)
+    await runSubProcess(
+      "pages generate features",
+      scope ? ["--scope" + " " + scope] : []
+    );
+
+  await createServer(!local, !!useProdURLs, scope);
+
+  if (openBrowser) await open(`http://localhost:${devServerPort}/`);
```

Comment:
The openBrowser argument still seems to not be passed through correctly

[JIRA ticket](https://yexttest.atlassian.net/browse/SUMO-5190?atlOrigin=eyJpIjoiYjQ5MTI3YjM1ZDAyNGYzODk4NDFmMWJmOTBkNTdiYjYiLCJwIjoiaiJ9)

Reply:
The openBrowser argument works. When running with npm run dev, the argument has to be passed to the inner command 'pages dev'. 

You can do so like this: `npm run dev -- --no-open-browser`